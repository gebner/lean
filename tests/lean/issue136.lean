
meta def tactic.interactive.test (a : interactive.parse
  (lean.parser.of_tactic (@tactic.fail â„• _ _ "oh no"))) :=
tactic.skip

example : true := begin test end -- should be "oh no"
