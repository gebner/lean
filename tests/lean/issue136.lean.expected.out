issue136.lean:6:29: error: oh no
issue136.lean:6:0: warning: declaration '[anonymous]' uses sorry
